src/schema.mjs

```diff
@@ -157,6 +157,43 @@ export const ipfs = {
   required: ["type", "commissioner", "version", "options"]
 };
 
+export const arweave = {
+  type: "object",
+  properties: {
+    type: {
+      type: "string",
+      enum: ["arweave"]
+    },
+    commissioner: {
+      type: "string"
+    },
+    version: {
+      type: "string"
+    },
+    options: {
+      type: "object",
+      properties: {
+        timeout: {
+          $comment: "temporal unit is milliseconds",
+          type: "integer"
+        },
+        uri: { 
+          type: "string",
+          pattern: "ar:\/\/"
```

Comment:
is there a regex for an arweave transaction?

Reply:
I think it is not exact

Reply:
I found other regex for it.
https://github.com/ArweaveTeam/arweave-deploy/blob/0770463bff7591890376651e54093fb66f3bb487/src/commands/send.ts#L5

Reply:
great, lol I've been looking for it too. Let's try to use it, what do you think?

Reply:
why can't we just use the property name "pattern"? From my understanding that's what we do anywhere.

Reply:
ah do you need the i flag?